lineapy/graph_reader/utils.py

```diff
@@ -15,3 +20,42 @@ def _is_import_node(graph: Graph, node_id: LineaID) -> bool:
                 lookup_node_name = lookup_node.__dict__.get("name", "")
                 return lookup_node_name in ["l_import", "getattr"]
     return False
+
+
+def get_db_artifacts_from_artifactdef(
+    db: RelationalLineaDB, artifact_entries: List[LineaArtifactDef]
+) -> List[LineaArtifact]:
+    # converts artifactdef list to a linea artfact list by fetching it from the db
+    return [
+        LineaArtifact.get_artifact_from_def(db, art_def)
+        for art_def in artifact_entries
+    ]
+
+
+def get_artifacts_grouped_by_session(
```

Comment:
I like the shorter and more concise:

group_artifacts_by_session( ... )

Reply:
agree, this relic is the original name.